app/views/posts/index.html.erb

```diff
@@ -1 +1,11 @@
-<h2> This is a test </h2>
\ No newline at end of file
+<section class='posts'>
+  <%= render'partials/user', user: @user %>
+    <% @posts.each do |post| %>
+   <a href="/users/<%= @user.id %>/posts/<%= post.id %>">
+    <ul class="post-list">
+    <%= render 'partials/post', post: post %></ul>
```

Comment:
Please use `#recent_five_comments` to render only last five comments on a post as it's requested by the previous code reviewer.